lib/Mount/MountProvider.php

```diff
@@ -149,7 +154,7 @@ private function getCurrentUID() {
 		return $user ? $user->getUID() : null;
 	}
 
-	public function getMount($id, $mountPoint, $permissions, $quota, $cacheEntry = null, IStorageFactory $loader = null, bool $acl = false, IUser $user = null): ?IMountPoint {
```

Comment:
Is the removal of the return type intentional?

Reply:
Not it isn't 😅 I will put it back